### PR TITLE
Apple home : Solo project

### PR DESCRIPTION
**Pull request by keddo**
### The main objective of the project is:
    - Building a web-page using images as a background and adding gradients to elements
    - The goal is to make a clone of old an old version of the Apple website web-page
### Finished project includes:
    - Add navbar at the top with gradient background
    - Add background about half of the page with header on it
    - Added footer

**Kindly request for review**

### CODE REVIEW

**Review by MarioDena — CHANGES_REQUESTED** [review]
Hello! Good job on your project! 👍  It looks great, however there are a few changes that needs to be implemented. 

## Changes required 

- [x] Please avoid using ID's for styling it is considered a bad practice. Implement your styles with classes only.

- [x] You live link on the read me is sending me to someone else's project.

- [x] There are repeated rules on your CSS please fix this, here is an article for help: https://speckyboy.com/good-bad-css-practices/

## Optional Changes

- [ ] Instead of using images for the text above the background you should use text, since an image is not maintainable. [Optional]

Cheers!  👍

**Comment by keddo** [comment]
- Used classes instead of ID's
 - Link to live project updated | I will use the actual live link after this project is approved instead of pointing out to github repo
 - Duplicate css rules got removed

**Comment by mikenath223** [comment]
### Pull request currently under review by Michgolden Ukeje (TSE).
Kindly exercise patience 👋

**Review by mikenath223 — CHANGES_REQUESTED** [review]
### Hi @keddo 👋

You've done quite well on this.👍
Kudos working on the requested changes in the previous review to get your project to this stage.👏

However, there remain a few more fixes to make your project look even more awesome:

- [x] Required - Kindly update the live link on your readme, sorry this wasn't mentioned in the previous review.

- [x] Required - Also as this project is focused on backgrounds and gradients please convert the four bottom images to backgrounds of their respective `div`s.[Screenshot]👇
ice_screenshot_20200421-094838

- [x] Required - In addition, please remove the inline comments that exists on both your HTML and CSS codes. It’s better to delete commented code. If you need to review a previous version of your code, use git history ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))

Kindly fix these issues and request another review👍
Feel free to contact me on slack(michgolden) if you need further clarification.

HAPPY CODING👨‍💻

**Comment by keddo** [comment]
- Kindly, I will update the live link after the code review so that I can deploy approved code to live server.
- Promo images changed to background image.
- inline comment code removed both from CSS and HTML code

**Comment by Dandush03** [comment]
# :red_circle: Status: Required Changes :red_circle:
### Hey :wave: 
### Good attempts on this one :muscle: :+1: 

## Your project looks great! but is a very important thing you have to consider before submitting another code review :point_down: 

- [x]  :one: You need to [Set-up Linter](https://github.com/microverseinc/linters-config/tree/master/css) in order to get this project approved.
  > I can see that you had added a stickler file but your PR is not running through the checks, I recommend you guys to check the [stickler](https://stickler-ci.com/) page to see if you had activated your project in there.

- [x] :two: Your Project is running into some merge conflicts that mos be resolved before getting this project approved. if you need any help please contact me privately and I'll help you get it done

###### Your PR should look like this at the bottom: 
Screenshot from 2020-04-01 09-19-32
###### Not like this
Screenshot from 2020-04-21 06-15-15

## Kindly address all these changes and submit a new code review

### Happy coding:computer: 

#### Daniel Laloush

**Comment by keddo** [comment]
### - Linter setup done
### - Fix merge conflicts

**Review by mikenath223 — CHANGES_REQUESTED** [review]
### Hi @keddo 👋

**You've done quite well on this project.👏**

- [x] Required - As requested in the previous review; kindly add a live link to your readme. Consider using this [githack tool](https://validator.w3.org/nu/#file) to create a live link.

- [x] Required - Also as requested in the previous review please remove all inline comments in both your HTML and CSS codes.

Kindly fix these issues and request another review👍
Feel free to contact me on slack(michgolden) if you need further clarification.

HAPPY CODING👨‍💻

File: index.html
Comment (mikenath223) [inline index.html]: Please remove this inline comment and other similar inline comments.

**Comment by keddo** [comment]
- Add Live link
- Remove inline comments

**Review by christianotieno — CHANGES_REQUESTED** [review]
Hello, @keddo Good job implementing some of the required changes. 
- Kindly, for purposes of review, always make sure that you are meeting all the criteria needed for approval.  Everything has to be implemented. Note that your PR-branch should always be treated as deployment-ready at all times. 
- The README file is not as descriptive as it expected. 
1. ADD a live link to your clone page.
2. Add a screenshot of the clone to your README as well. It's not showing currently.
##
Submit another code review request once your branch is deployment-ready and you have fixed the required changes. 
Cheers
[Christian Otieno](https://github.com/christianotieno)

**Comment by keddo** [comment]
- screenshot added
- live link added

**Comment by mikenath223** [comment]
This review request has been marked as invalid.

### STATUS: INVALID REQUEST ✖

Kindly send a review request using the right pull request link that hasn't been merged, as this seems to have been merged already.

**Pull request merged**
